src/main.c

```diff
@@ -2437,6 +2442,12 @@ void handle_movement(double dt) {
                 dy = 8;
             }
         }
+        /**This condition is used to implement logic for instant high fly feature.*/
```

Comment:
This comment is somewhat too complex & confuses me with line 2254. I think a better more straight forward comment would be "Fly high".

Reply:
change done.